### PR TITLE
Editing new Setup Wiz chapter in Install Guide

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/uyuni-project/uyuni-docs/pull/66

### CODE REVIEW

File: modules/installation/pages/setup-wizard.adoc [hunk 1]
```diff
@@ -1,29 +1,36 @@
-[[ref.webui.admin.wizard]]
+[[Setrup Wizard]]
```
Comment (keichwa) [inline modules/installation/pages/setup-wizard.adoc]:
```suggestion
[[Setup Wizard]]
```
I guess we must clarify how these label should look. What will be the end result (file names of the HTML output).

File: modules/installation/pages/setup-wizard.adoc [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
+
+
+== Configure Organization Credentials
+
+Your {scc} account is associated with the administration account of your organization.
+You can share your {scc} access with other users within your organization.
+Navigate to the [guimenu]``Organization Credentials`` tab to grant users within your organization access to your {scc} account.
+
+Click btn:[Add a new credential], enter the username and password of the user to grant access to, and click btn:[Save].
+A new credential card is shown for the user your have granted access to.
```
Comment (keichwa) [inline modules/installation/pages/setup-wizard.adoc]:
```suggestion
A new credential card is shown for the user you have granted access to.
```

File: modules/installation/pages/setup-wizard.adoc [hunk 3]
```diff
@@ -1,29 +1,36 @@
...
-+
 
-menu:Organization Credentials:[]::
-Select menu:Admin[Setup Wizard > Organization Credentials > Add a new credential] then enter user name and password to give another organization/user access to {scc}.
-+
+== Configure the HTTP Proxy
 
-image::admin_organization_credentials.png[scaledwidth=80%]
-+
+{productname} can connect to the {scc} or other remote servers using a proxy.
```
Comment (keichwa) [inline modules/installation/pages/setup-wizard.adoc]:
```suggestion
{productname} can connect to the {scc} (SCC) or other remote servers using a proxy.
```
Later on, we use just "SCC".  I think it's good practice to define the abbrev this way where it occurs for the first time in a chapter or section.

File: modules/installation/pages/setup-wizard.adoc [hunk 4]
```diff
@@ -34,77 +41,29 @@ To re-check the credential with SCC, click the icon.
...
-Once a product has finished downloading, the status bar state will change from a filled percentage value to menu:SUSE Products[Finished].
+== Configure Products
+
+
+Your {suse} subscription entitles you to access a range of products.
+Navigate to the [guimenu]``Products`` tab to browse the products available to you and synchronize {productname} with {scc}.
+
+Filters help you search for products by description or architecture.
+
+The list is organised by product name.
```
Comment (keichwa) [inline modules/installation/pages/setup-wizard.adoc]:
```suggestion
The list is organized by product name.
```
;)

Comment (Loquacity) [reply modules/installation/pages/setup-wizard.adoc]: :woman_facepalming:

File: modules/installation/pages/setup-wizard.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/installation/pages/setup-wizard.adoc]: oh, it probably needs a hyphen, too.

**Pull request merged**